recipes/metview/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "5.6.0" %}
+
+package:
+  name: metview
+  version: {{ version }}
+
+source:
+  url: https://confluence.ecmwf.int/download/attachments/30048389/Metview-{{ version }}-dev3-Source.tar.gz
```

Comment:
An attachment in confluence seems hard to update, specially because of that id `30048389` part. It would be better if it was a link to PyPI or to a GitHub or Bitbucket tagged release, for example.

Reply:
I believe the id number is a page id - it's constant as long as the tarballs are all attached to the same page on confluence (which they are at ECMWF). Certainly the other ECMWF feedstocks have been using confluence like this without issue!